### ISSUE TITLE
Create redcode repository and hall of fame

### ISSUE THREAD

**Issue by gareththegeek** [gareththegeek/corewar#32]
Host a database of redcode warriors.  When you are writing your own warrior, the warrior database will give your warrior something to fight against.

The database could also record stats for the warrior and allow us to host a ranked hall of fame

**Comment by dougajmcdonald** [comment]
Definitely! we should allow people to select from `base` warriors which would have a description of what they do and how the commands do that, as part of a tutorial. Encourage people to modify existing warriors rather than always needing to start from scratch

**Comment by gareththegeek** [comment]
I think this should go to the UI / Server project if it hasn't already
